lib/core-integration/src/Test/Integration/Scenario/API/ByronWallets.hs

```diff
@@ -188,7 +189,53 @@ spec = do
                     getFromResponse amount r0
             actualFee `shouldBe` predictedFee
```

Comment:
I'd instead look for the following assertion:

```hs
        -- Wait for the funds to have arrived onto the new wallet
        eventually $ do
            rNew <- request @ApiWallet ctx (getWalletEp newW) Default Empty
            verify rNew
                [ expectFieldEqual balanceAvailable (faucetAmt - fee)
                , expectFieldEqual balanceTotal (faucetAmt - fee)
                ]
```

Which is stronger as it shows that the estimated fee were correct and that funds have arrived on the target wallet.

Reply:
@KtorZ That stronger test is available in PR #960.